store.html

```diff
@@ -0,0 +1,154 @@
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Storage</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/store.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="icon" href="img/karma-logo.svg" />
+  </head>
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="logo" />
+      <nav>
+        <ul>
+          <li><a href="index.html">Meet Karma</a></li>
+          <li><a href="#">How it Works</a></li>
+          <li><a href="store.html">Store</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="left-section">
+        <form>
+          <h1>Order your Karma wifi device today!</h1>
+          <fieldset class="name-fieldset">
+            <div>
+              <label for="first-name">First name * </label>
+              <input name="first_name" type="text" id="first-name" required />
+            </div>
+            <div>
+              <label for="last-name">Last name * </label>
+              <input name="last_name" type="text" id="last-name" required />
+            </div>
+          </fieldset>
+          <fieldset class="address-fieldset">
+            <div>
+              <label for="address">Address *</label>
+              <input name="address" type="text" id="address" required />
+            </div>
+            <div>
+              <label for="address2">Address 2</label>
+              <input name="address2" type="text" id="address2" />
+            </div>
+            <div class="city-postcode-div">
+              <div class="city-div">
+                <label for="city-input">City *</label>
+                <input
+                  list="city"
+                  name="city-input"
+                  id="city-input"
+                  placeholder="Select your city..."
+                />
+                <datalist name="city" id="city">
+                  <option value="Barcelona">Barcelona</option>
+                  <option value="Madrid">Madrid</option>
+                  <option value="Valencia">Valencia</option>
+                  <option value="Girona">Girona</option>
+                  <option value="Malaga">Malaga</option>
+                </datalist>
+              </div>
+              <div class="postcode-div">
+                <label for="postcode">Postcode *</label>
+                <input type="text" name="postcode" id="postcode" required />
+              </div>
+            </div>
+          </fieldset>
+          <fieldset class="color-fieldset">
+            <p>Select a color</p>
+            <div class="color-radio-div">
```

Comment:
I would remove the `margin-bottom` here, it's quite a lot of space.

Reply:
I did a bottom margin here because it is consistent with sketch I should follow

Reply:
Ok, then it's fine. I was just wondering if its too much space.

🟢 Feel free to resolve.